modules/veteran/spec/models/veteran/user_spec.rb

```diff
@@ -9,7 +9,7 @@
     it 'initializes from a user' do
       VCR.use_cassette('bgs/claimant_web_service/find_poa_by_participant_id') do
         allow_any_instance_of(BGS::OrgWebService).to receive(:find_poa_history_by_ptcpnt_id)
-          .and_return({ person_poa_history: { person_poa: [{ begin_date: Time.zone.now, legacy_poa_cd: '033' }] } })
+          .and_return({ person_poa_history: { person_poa: [{ begin_dt: Time.zone.now, legacy_poa_cd: '033' }] } })
```

Comment:
i spent an embarrassing amount of time thoroughly confused why the sort wasn't working locally before i discovered this key was incorrect :(